### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by io7m** [io7m/kstructural#15]
Add the appropriate accessibility attributes to all generated elements (such as `title` attributes on all generated links).

**Comment by io7m** [comment]
This seems complete. Missing functionality should be in new tickets.
